src/components/js/EcAppCard.js

```diff
@@ -1,37 +1,53 @@
 import { i18n } from '@ecomplus/utils'
-import { i19install, i19edit } from '@ecomplus/i18n'
 
-const MAX_LENGHT_DESCRIPTION = 100
+import {
+  i19free
+  // i19paid
+} from '@ecomplus/i18n'
 
 export default {
   name: 'EcAppCard',
+
   props: {
     app: {
       type: Object,
       default: {}
     },
-    isInstalled: {
-      type: Boolean
+    descriptionMaxLength: {
+      type: Number,
+      default: 95
     }
   },
+
   computed: {
-    i19edit () {
-      return i18n(i19edit)
+    i19free () {
+      return i18n(i19free)
     },
-    i19install () {
-      return i18n(i19install)
+
+    i19paid () {
+      return 'Pago'
     },
-    appUrlIcon () {
-      if (!this.app.icon.startsWith('https://')) {
-        return `https://market.e-com.plus${this.app.icon}`
+
+    iconUrl () {
+      if (this.app.icon) {
+        if (!this.app.icon.startsWith('https://')) {
+          return `https://market.e-com.plus${this.app.icon}`
+        }
+        return this.app.icon
       }
-      return this.app.icon
+      return ''
     },
+
     formattedDescription () {
-      if (this.app.short_description.length > MAX_LENGHT_DESCRIPTION) {
-        return `${this.app.short_description.slice(0, MAX_LENGHT_DESCRIPTION)}...`
+      if (this.app.short_description) {
+        const { descriptionMaxLength } = this
+        if (this.app.short_description.length > descriptionMaxLength) {
+          return `${this.app.short_description.slice(0, descriptionMaxLength)}...`
+        }
+        return this.app.short_description
+      } else {
```

Comment:
@leomp12 Acho que não precisa do else aqui, já poderia retornar direto se não entrar no `if (this.app.short_description)`

Reply:
Boa!

Reply:
boa @thiagoabreudev